changelog/@unreleased/pr-948.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: Dialogue clients use a sympathetic validation interval when servers
+    explicitly set a `Keep-Alive` response header including a timeout duration.
```

Comment:
:+1: this functionality is available in WC 0.289.4 https://internal-github/foundry/witchcraft/pull/2640, but servers will havee to opt in using a install.server config option